src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -569,16 +557,20 @@ export const TradeInput = memo(({ isCompact }: TradeInputProps) => {
       receiveAddress,
       walletSupportsBuyAssetChain,
       maybeUnsafeTradeWarning,
-      isUnsafeQuote,
-      isUnsafeQuoteNoticeDismissed,
-      handleAcknowledgeUnsafeQuote,
       quoteHasError,
       shouldDisablePreviewButton,
       quoteStatusTranslation,
     ],
   )
 
   const handleFormSubmit = useMemo(() => handleSubmit(onSubmit), [handleSubmit, onSubmit])
+  const handleTradeQouteConfirm = useCallback(() => {
+    if (isUnsafeQuote) {
+      return setShouldShowWarningAcknowledgement(true)
+    } else {
```

Comment:
Ditch the `else` to avoid squirly braces, or don't return above if we're keeping it

Reply:
Addressed in https://github.com/shapeshift/web/pull/6647/commits/916ecb47d6f7964089d596f97a9406fbe95c1d43.